.azurePipeline/templatePublishLogsFromPods.yml

```diff
@@ -0,0 +1,30 @@
+# Template to login into Dockerhub, build a docker image and push it.
+
+# Parameters:
+# `logsFolderPath` is the folder where the logs from the pods will be stored.
+# `releaseName` is the name of the release used to get the relevant list of pods.
+# `kubeConfigFile` is the kubeConfigFile path.
+
+parameters:
+  logsFolderPath: ''
+  releaseName: ''
+  kubeConfigFile: ''
+
+steps:
+- task: UsePythonVersion@0
+  inputs:
+    versionSpec: '3.7'
+  condition: failed()
+- script: |
+    echo "Copy the pods' logs from the release '${{ parameters.releaseName }}' to '${{ parameters.logsFolderPath }}'."
+    python .azurePipeline/getPodsLogs.py ${{ parameters.logsFolderPath }} ${{ parameters.releaseName }}
+  displayName: 'Copy the logs from the service pods.'
+  condition: failed()
+  env:
+    KUBECONFIG: ${{ parameters.kubeConfigFile }}
+- task: PublishPipelineArtifact@1
```

Comment:
I'm guessing you didn't want to just `cat` them all into the Azure Devops output log? It would make it even easier to diagnose a test failure...

Reply:
I don't really like looking at logs from the Azure Dev-ops page, which is why I tend to prefer files I can view with the tools I prefer. If people prefer from the Azure dev-ops page, we can indeed `cat` the files, but will need to choose a preferred way to split the output in a nice way.

Reply:
Fair enough!